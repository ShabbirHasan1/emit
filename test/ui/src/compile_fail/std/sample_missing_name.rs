fn main() {
    emit::sample!(value: 42);
}
