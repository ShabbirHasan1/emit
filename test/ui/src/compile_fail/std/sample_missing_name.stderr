error: either `name` needs to be specified, or `value` must be an identifier to infer `name` from, like: `let my_metric = 42; emit::sample!(value: my_metric);`
 --> src/compile_fail/std/sample_missing_name.rs:2:26
  |
2 |     emit::sample!(value: 42);
  |                          ^^
